website/sidebars.js

```diff
@@ -246,6 +246,7 @@ module.exports = {
                         'user_guide/activation_strategy',
                         'advanced/archived_toggles',
                         'advanced/audit_log',
+                        'advanced/impression_data',
```

Comment:
We should move this to between projects and role-based access control to keep the list sorted 🤓

Reply:
Fair!